tensorrt_llm/_torch/attention_backend/sparse/minimax_m3/msa_backend.py

```
@@ -0,0 +1,789 @@
+    require_msa_module,
+)
+
+
+def _cache_device(meta) -> torch.device:
+    """Device hosting the paged KV buffers, else the current CUDA device."""
+    kv_cache_manager = meta.kv_cache_manager
+    if kv_cache_manager is not None:
+        try:
+            return kv_cache_manager.get_buffers(0).device
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)